### ISSUE TITLE
cannot import TimeSeries

### ISSUE THREAD

**Issue by javier-iaa** [gwpy/gwpy#582]
I have no idea how to fix this. Can you help me?

```
>>> from gwpy.timeseries import TimeSeries
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/gwpy/timeseries/__init__.py", line 22, in <module>
    from .core import *
  File "/usr/local/lib/python2.7/dist-packages/gwpy/timeseries/core.py", line 52, in <module>
    from ..types import (Array2D, Series)
  File "/usr/local/lib/python2.7/dist-packages/gwpy/types/__init__.py", line 26, in <module>
    from .array import *
  File "/usr/local/lib/python2.7/dist-packages/gwpy/types/array.py", line 38, in <module>
    from ..detector import Channel
  File "/usr/local/lib/python2.7/dist-packages/gwpy/detector/__init__.py", line 28, in <module>
    from .channel import *
  File "/usr/local/lib/python2.7/dist-packages/gwpy/detector/channel.py", line 38, in <module>
    from ..io import (datafind, nds2 as io_nds2)
  File "/usr/local/lib/python2.7/dist-packages/gwpy/io/nds2.py", line 352, in <module>
    sample_rate=None, type=Nds2ChannelType.any(),
  File "/usr/local/lib/python2.7/dist-packages/gwpy/io/nds2.py", line 69, in any
    return reduce(operator.or_, (x.value for x in cls))
TypeError: 'type' object is not iterable
 ```

**Comment by duncanmmacleod** [comment]
@javier-iaa, apologies for the delay, can you post the version of gwpy you installed? Also, please post the operating system version.

**Comment by javier-iaa** [comment]
Hi Duncan,
Thank you for your response. I installed gwpy version 0.6.2 using pip on a Point Linux distribution which is based on the latest Debian 3.16.51-2. I run this OS through a virtual machine using VMVirtualBox 5.1.30.

**Comment by duncanmmacleod** [comment]
@javier-iaa, I think I have nailed down the problem. In 0.6.2, gwpy doesn't declare its dependency on `enum34` (fixed in #594), so users who run `pip install gwpy` probably end up seeing the warning:

```python
>>> from gwpy.timeseries import TimeSeries
ImportError: No module named enum
```

The natural response is then to run `pip install enum`. However [`enum`](https://pypi.python.org/pypi/enum) != [`enum34`](https://pypi.python.org/pypi/enum34), with the former being incompatible with gwpy.

So, what _you_ need to do is

```pip uninstall enum -yq && pip install enum34```

or wait until gwpy 0.7 is released, which should hopefully be sometime in the next few weeks.

Additionally, you will likely need to install [`ligotimegps`](https://pypi.python.org/pypi/ligotimegps) to get the desired import to work, since that dependency is also not declared properly in 0.6.2.

Please confirm that this resolves your issue.

**Comment by javier-iaa** [comment]
Yes, that solve the issue definitively. Thanks a lot!
